web-crawler/src/test/java/com/google/sps/WebCrawlerTest.java

```diff
@@ -14,7 +14,7 @@
 
 import com.google.sps.webcrawler.WebCrawler;
 
-public class WebCrawlerTestor {
+public class WebCrawlerTest {
 
   public static void main(String[] args) {
```

Comment:
Usually unit tests don't have a main method, but I realize you had different intentions for this class.

Reply:
I haven't started implementing the unit tests. I'll try to get to those soon